lib/gettext/plural.ex

```diff
@@ -256,282 +256,25 @@ defmodule Gettext.Plural do
-    "ha",
-    # Hebrew
-    "he",
-    # Hindi
-    "hi",
-    # Chhattisgarhi
-    "hne",
-    # Armenian
-    "hy",
-    # Hungarian
-    "hu",
-    # Interlingua
-    "ia",
-    # Italian
-    "it",
-    # Greenlandic
-    "kl",
-    # Kannada
-    "kn",
-    # Kurdish
-    "ku",
-    # Letzeburgesch
-    "lb",
-    # Maithili
-    "mai",
-    # Malayalam
-    "ml",
-    # Mongolian
-    "mn",
-    # Manipuri
-    "mni",
-    # Marathi
-    "mr",
-    # Nahuatl
-    "nah",
-    # Neapolitan
-    "nap",
-    # Norwegian Bokmal
-    "nb",
-    # Nepali
-    "ne",
-    # Dutch
-    "nl",
-    # Northern Sami
-    "se",
-    # Norwegian Nynorsk
-    "nn",
-    # Norwegian (old code)
-    "no",
-    # Northern Sotho
-    "nso",
-    # Oriya
-    "or",
-    # Pashto
-    "ps",
-    # Punjabi
-    "pa",
-    # Papiamento
-    "pap",
-    # Piemontese
-    "pms",
-    # Portuguese
-    "pt",
-    # Romansh
-    "rm",
-    # Kinyarwanda
-    "rw",
-    # Santali
-    "sat",
-    # Scots
-    "sco",
-    # Sindhi
-    "sd",
-    # Sinhala
-    "si",
-    # Somali
-    "so",
-    # Songhay
-    "son",
-    # Albanian
-    "sq",
-    # Swahili
-    "sw",
-    # Swedish
-    "sv",
-    # Tamil
-    "ta",
-    # Telugu
-    "te",
-    # Turkmen
-    "tk",
-    # Urdu
-    "ur",
-    # Yoruba
-    "yo"
-  ]
-
-  @two_forms_2 [
-    # Acholi
-    "ach",
-    # Akan
-    "ak",
-    # Amharic
-    "am",
-    # Mapudungun
-    "arn",
-    # Breton
-    "br",
-    # Filipino
-    "fil",
-    # French
-    "fr",
-    # Gun
-    "gun",
-    # Lingala
-    "ln",
-    # Mauritian Creole
-    "mfe",
-    # Malagasy
-    "mg",
-    # Maori
-    "mi",
-    # Occitan
-    "oc",
-    # Tajik
-    "tg",
-    # Tigrinya
-    "ti",
-    # Tagalog
-    "tl",
-    # Turkish
-    "tr",
-    # Uzbek
-    "uz",
-    # Walloon
-    "wa"
-  ]
-
-  @three_forms_slavic [
-    # Belarusian
-    "be",
-    # Bosnian
-    "bs",
-    # Croatian
-    "hr",
-    # Serbian
-    "sr",
-    # Russian
-    "ru",
-    # Ukrainian
-    "uk"
-  ]
-
-  @three_forms_slavic_alt [
-    # Czech
-    "cs",
-    # Slovak
-    "sk"
-  ]
-
   # Default implementation of the init/1 callback, in case the user uses
   # Gettext.Plural as their plural forms module.
   @doc false
   def init(context)
 
   def init(%{locale: locale, plural_forms_header: plural_forms_header}) do
-    with "nplurals=" <> rest <- String.trim(plural_forms_header),
-         {plural_forms, _rest} <- Integer.parse(rest) do
-      {locale, plural_forms}
-    else
-      _other -> locale
+    case Expo.PluralForms.parse(plural_forms_header) do
+      {:ok, plural_forms} ->
+        {locale, plural_forms}
+
+      {:error, _reason} ->
+        # Fall back to parsing headers such as "nplurals=3", without the "plural=..." part.
+        # TODO: remove this at some point.
+        with "nplurals=" <> rest <- String.trim(plural_forms_header),
+             {plural_forms, _rest} <- Integer.parse(rest) do
+          {locale, plural_forms}
```

Comment:
Would it make sense to put something along the lines of this (untested pseudo-code):

```suggestion
          case Expo.PluralForms.plural_form(locale) do
            {:ok, plural_form} -> IO.warn("""
            Deprecated `Plural-Forms` header found:
            Current: #{inspect(plural_forms_header)}
            Recommendation: #{inspect(Expo.PluralForms.to_string(plural_form))}
            """)
            :error -> :ok
          end
          {locale, plural_forms}
```

Reply:
Yes! I'll do this in another PR, where we can deprecate `--plural-forms` as well and so on.